airflow/executors/workloads.py

```diff
@@ -50,6 +50,10 @@ class TaskInstance(BaseModel):
     try_number: int
     map_index: int | None = None
 
+    pool_slots: int
+    queue: str
+    priority_weight: int
```

Comment:
@kaxil Should this be here, or in the context returned from the `/run` endpoint?

Reply:
At least that are attributes on TaskInstance in the ORM model. Just adding these three lines filled it over. No objection moving it somewhere else but as here it is consistent with the backend.

Reply:
I think it will be needed for the executor to route to correct `pool` before the task is marked as `queued`